server.js

```diff
@@ -41,15 +41,17 @@ global.mongoose = mongoose;
 const app = express();
 
 const allowCrossDomain = function (req, res, next) {
-    if (process.env.NODE_ENV !== "production") {
+    if (process.env.NODE_ENV && process.env.NODE_ENV.toLowerCase() !== "production") {
         res.header("Access-Control-Allow-Origin", "http://localhost:3000");
     }
     res.header("Access-Control-Allow-Headers", "Content-Type, Authorization");
     res.header("Access-Control-Allow-Credentials", true);
+    res.header("Access-Control-Allow-Origin", "https://dibs-vancouver.herokuapp.com");
```

Comment:
I think this isn't necessary if the requests are coming from the same origin. This will also reject localhost requests

Could you push another commit to delete this line?